rackspace/metrics/v2/ingestion/requests.go

```diff
@@ -0,0 +1,205 @@
+package ingestion
+
+import (
+	"github.com/rackspace/gophercloud"
+	"fmt"
+)
+
+// SendMetrics will ingest the metrics for the tenant associated with RackspaceMetrics.
+func SendMetrics(c *gophercloud.ServiceClient, metrics []MetricData) {
+	var res PostResult
+	reqBody := make([]map[string]interface{}, len(metrics))
+	for i := range metrics {
+		Metric := metrics[i]
+		reqBody[i] = map[string]interface{}{
+			"collectionTime": Metric.CollectionTime,
+			"ttlInSeconds":Metric.TtlInSeconds,
+			"metricValue": Metric.MetricValue,
+			"metricName": Metric.MetricName,
```

Comment:
Thanks @sam-falvo for helping out! I now understood how to directly decode/encode structure to map or vice-versa using "mapstructure". So for the fixed fields I'll be using that and will try to remove as many redundancies as possible. Thanks @rgbkrk for your patience :smiley: